cargo-shuttle/src/lib.rs

```diff
@@ -152,7 +167,23 @@ impl Shuttle {
             Command::Login(login_args) => self.login(login_args).await,
             Command::Logout(logout_args) => self.logout(logout_args).await,
             Command::Feedback => self.feedback().await,
-            Command::Run(run_args) => self.local_run(run_args).await,
+            Command::Run(mut run_args) => {
+                let range_end = 65535;
+                if !portpicker::is_free_tcp(run_args.port) {
+                    let available_port = Shuttle::find_available_port(run_args.port - 1, range_end);
```

Comment:
Why do we start checking from `port - 1`? Would it be better if we check from `port + 1` instead and remove +1 from `available_port` in `find_available_port` function?

Reply:
updated the logic and incorporated @jonaro00's suggestion too.

Reply:
to increase the base port by 10